scapy/packet.py

```diff
@@ -1725,6 +1736,9 @@ def explore(layer=None):
                     result = result_contrib
                 else:
                     raise Scapy_Exception("Unknown scapy module '%s'" % layer)
+        else:
+            raise Scapy_Exception("Wrong usage ! Check out help(explore)")
```

Comment:
I think that printing the error message is enough.

Reply:
I don’t get it :/ you mean we shouldn’t error when the format isn’t correct ?

In your example previously, you mistaken explore for ls: explore is to search on layers (by file name or module object), whereas ls search packets (by name or class). If we don’t raise an error, people are probably going to mistake them the same way you did :/

Reply:
Previously, the function was crashing. If I am correct, your fix now raise `Scapy_Exception` to prevent this crash.

I think that in interactive mode, it is more "user friendly" to print an error message, rather than raising an exception. It is less scary= )